src/services/iasql.ts

```diff
@@ -731,6 +723,16 @@ export async function install(moduleList: string[], dbAlias: string, user: any,
       i--;
     }
   }
+  // TODO rm special casing for aws_account
+  // Check to make sure that all dependent modules are in the list
+  const missingDeps = mods
+    .flatMap((m: Modules.ModuleInterface) => m.dependencies.filter(d => !moduleList.includes(d) && !existingModules.includes(d)))
```

Comment:
added the check for `existingModules`

Reply:
meaning the ones are already installed, right?

Reply:
yes